src/index.js

```diff
@@ -36,9 +36,14 @@ module.exports = function lintStaged(injectedLogger, configPath) {
     .then(result => {
       if (result == null) throw errConfigNotFound
 
+      const configResult = {
+        verbose,
+        ...result.config
```

Comment:
We are not transpiling the source code and destructuring objects isn't supported in node v4 and v6. Additionally, at this point the `result.config` isn't normalized. So it could be in the simple or advanced format. You can refer https://github.com/okonet/lint-staged#configuration to know more about configuration. If the type is simple, then adding the verbose flag to it would be incorrect.